stanford-earth/SE3_BLT
# EARTH-409: Added simplesamlphp and externalauth modules with split enable

# READY FOR REVIEW

# Summary
- Adds two new modules just for production
- Fixes up some config split settings

# Steps to Test

1. Check out this branch
2. Run `drush cim` to import config 
3. Ensure the simpesamlphp_auth module is still disabled
4. Run `drush csim prod -y` 
5. Ensure the simpesamlphp_auth and externalauth modules are enabled

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
